### DISCUSSION TITLE
2 high severity vulnerabilities

### DISCUSSION THREAD

**Discussion by andluk01** [jef/streetmerchant#2261] [Help]
when i try to run the comand "npm instal" appears this message:

up to date, audited 1184 packages in 8s

2 high severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.

what can I do solve this problem?

**Comment by 0000day** [comment]
Everyone got this message. It's code/dependency related. If the vulnerabilities apply to this app is questionable.

If _you_ want to do something about it locally, I would suggest doing what's suggested: `npm audit`. Check the type of vulns, check for updates, check for deprecated packages, maybe lookup alternativ packages.
